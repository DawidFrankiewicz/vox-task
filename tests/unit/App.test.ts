import { mount } from '@vue/test-utils';
import { test, expect } from 'vitest';

const factory = async () => {
    const component = await import('@/App.vue');

    return mount(component.default, {
        shallow: true,
    });
};

test('Should render correctly', async () => {
    const wrapper = await factory();

    // Check if component matches snapshot (HTML structure)
    // https://vitest.dev/guide/snapshot#updating-snapshots
    expect(wrapper.html()).toMatchSnapshot();
});
