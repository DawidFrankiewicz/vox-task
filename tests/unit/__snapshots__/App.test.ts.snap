// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render correctly 1`] = `"<blocks-board-stub></blocks-board-stub>"`;
